package/MDAnalysis/analysis/nucleicacids.py

```diff
@@ -183,50 +290,213 @@ class WatsonCrickDist(NucPairDist):
 
     Attributes
     ----------
-    times: numpy.ndarray
-        Simulation times for analysis.
     results.pair_distances: numpy.ndarray
         2D array of Watson-Crick basepair distances. First dimension is
         simulation time, second dimension contains the pair distances for
         each each entry pair in strand1 and strand2.
 
         .. versionadded:: 2.4.0
 
+    results.distances: numpy.ndarray
+        stored in a 2d numpy array with first index selecting the Residue pair, 
+        and the second index selecting the frame number
+    times: numpy.ndarray
+            Simulation times for analysis.
 
     Raises
     ------
-    ValueError
-        If the residues given are not amino acids
+    DeprecationWarning
+        If a lits of :class:`~MDAnalysis.core.groups.Residue` is given for
+        :attr:`strand1` of :attr:`strand2` instead of a
+        :class:`~MDAnalysis.core.groups.ResidueGroup`
     ValueError
         If the selections given are not the same length
-
+    ValueError:
+        An :class:`~MDAnalysis.core.groups.AtomGroup` in one of the strands not
+        a valid nucleic acid
+    ValueError
+        If a given residue pair from the provided strands returns an empty
+        :class:`~MDAnalysis.core.groups.AtomGroup` when selecting the atom
+        pairs used in the distance calculations
 
     .. versionchanged:: 2.5.0
        Accessing results by passing strand indices to :attr:`results` is
        was deprecated and is now removed as of MDAnalysis version 2.5.0. Please
        use :attr:`results.pair_distances` instead.
        The :attr:`results.times` was deprecated and is now removed as of
        MDAnalysis 2.5.0. Please use the class attribute :attr:`times` instead.
+
+    .. versionchanged:: 2.7.0
+        Deprecated the use of :attr:`List[Residue]` instead use
+        :class:`~MDAnalysis.core.groups.ResidueGroup`.
```

Comment:
This is a bit tricky; normally I would put a `.. deprecated::` role here, but that's going to be too confusing for the casual reader. We're really only deprecating one type of input so I'd rewrite as:
```suggestion
        `strand1` and `strand2` now also accept a :class:`~MDAnalysis.core.groups.ResidueGroup`
        as input. The previous input type of ``List[Residue]`` is still supported
        but it is **deprecated** and will be removed in release 3.0.0.
```

Reply:
done